server/src/test/java/org/elasticsearch/index/mapper/TimeSeriesIdFieldMapperTsidHashingTests.java

```diff
@@ -0,0 +1,676 @@
+/*
```

Comment:
Maybe instead of duplicating this test, just change TimeSeriesIdFieldMapperTests to do hashing. In order to test the older tsid behaviour just add one or two tests that use createMapperService(...) that accepts a concrete version.

Reply:
The same I said above applies here I think...the test is just a duplicate (which I added on the first place because my original plan was to have a test for "before" the change and one for "after" the change (change = introduction of tsid hashing).